dev-requirements.txt

```diff
@@ -20,4 +20,4 @@ scikit-learn~=1.0
 lightgbm~=3.0; platform_machine != 'arm64'
 xgboost~=1.1
 sqlalchemy_utils~=0.39.0
-import-linter~=1.8
\ No newline at end of file
+import-linter~=1.8
```

Comment:
what changed here? 🤔

Reply:
newline

Reply:
new line (or lack of...) vi changes that automagically